dts/sdm450-samsung-r05.dts

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8953.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <351 0x0>;
+	qcom,board-id = <8 5>;
+	
+	gta2xlwifi-generic {
+		model = "Samsung Tab A2 XL WIFI Rev05";
+		compatible = "samsung,gta2xlwifi-eur", "qcom,sdm450", "qcom,msm8953", "lk2nd,device";
+		lk2nd,match-bootloader = "T590*";
+		lk2nd,pstore = <0x86100000 0x100000>;
+	};
+	
+	// Bootloader won't continue if it can't delete some nodes from below
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges = <0x0 0x0 0x0 0xffffffff>;
+
+		qcom,memshare {
+			qcom,client_4 {
+				qcom,peripheral-size = <0x6000000>;
+			};
+		};
+	};
+
+	reserved-memory {
```

Comment:
need to delete some nodes, or the kernel will not be able to start because the memory is already being allocated